<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomSizeSetter</class>
 <widget class="QDialog" name="CustomSizeSetter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter new measure parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QRadioButton" name="areaRadioButton">
     <property name="text">
      <string>Area</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QRadioButton" name="distanceRadioButton">
     <property name="text">
      <string>Distance</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Combo box text</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="comboTextLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="measureLabel">
     <property name="text">
      <string>Area in mmÂ²</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="measureLineEdit"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="okPushButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="cancelPushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
